src/Distributed/interleave_comm_and_comp.jl

```diff
@@ -0,0 +1,63 @@
+using Oceananigans: prognostic_fields
+using Oceananigans.Grids: halo_size
+
+function complete_communication_and_compute_boundary!(model, ::DistributedGrid, arch)
+
+    # We iterate over the fields because we have to clear _ALL_ architectures
+    # and split explicit variables live on a different grid
```

Comment:
```suggestion
    # We iterate over the fields because we need to clear _ALL_ architectures
    # and split-explicit variables live on a different grids
```